src/ontology/cl-edit.owl

```diff
@@ -31484,6 +31487,36 @@ AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:27776108") oboInOwl:hasE
 AnnotationAssertion(rdfs:label obo:CL_4033040 "lung resident memory CD8-positive, CD103-positive, alpha-beta T cell")
 EquivalentClasses(obo:CL_4033040 ObjectIntersectionOf(obo:CL_4033039 ObjectSomeValuesFrom(obo:RO_0002104 obo:PR_000001010)))
 
+# Class: obo:CL_4033041 (alveolar macrophage CCL3-positive)
```

Comment:
Labels in CL tend to have markers at the start of the label, so consider relabelling to 'CCL3-positive alveolar macrophage'.